corehq/blobs/migrate.py

```diff
@@ -414,23 +451,26 @@ class BackendMigrator(Migrator):
 
     has_worker_pool = True
 
-    def __init__(self, slug, doc_migrator_class):
-        reindexer = BlobMetaReindexAccessor()
+    def __init__(self, slug, doc_migrator_class, type_code=None):
+        reindexer = BlobMetaReindexAccessor(type_code=type_code)
         types = [reindexer.model_class]
         assert not hasattr(types[0], "get_db"), types[0]  # not a couch model
         super(BackendMigrator, self).__init__(slug, types, doc_migrator_class)
         self.reindexer = reindexer
 
-    def get_doc_migrator(self, filename, date_range=None, **kw):
+    def get_doc_migrator(self, filename, **kw):
         migrator = super(BackendMigrator, self).get_doc_migrator(filename)
         return _migrator_with_worker_pool(migrator, self.reindexer, **kw)
 
-    def get_document_provider(self, date_range=None):
+    def get_document_provider(self, date_range=None, domain=None):
         iteration_key = self.iteration_key
         if date_range:
             (start, end) = date_range
             self.reindexer.date_range = date_range
-            iteration_key = '{}-{}-{}'.format(self.iteration_key, start, end)
+            iteration_key = f'{self.iteration_key}-{start}-{end}'
+        if domain:
+            self.reindexer.domain = domain
+            iteration_key = f'{self.iteration_key}-{domain}'
```

Comment:
```suggestion
            iteration_key = f'{iteration_key}-{domain}'
```

Reply:
I'm assuming the previous suggestion I left here was resolved by mistake because it looked so much like the other one. I think it still needs to be changed. If not, can you explain why not?
```suggestion
            iteration_key = f'{iteration_key}-{domain}'
```